proj/src/middleware/projectsMiddleware.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const projectsMiddleware = ({dispatch}) => next => action => {
+  if (!action.meta || action.meta.type !== 'api') {
```

Comment:
wouldn't a check for meta.api be enough here?

Reply:
an action*